app/client/components/filterModule/FilterModule.jsx

```diff
@@ -94,11 +94,13 @@ class FilterModule extends React.Component {
     if (groupBy) {
       params.group_by = groupBy;
     }
-
-    if (dateRange) {
+    if (dateRange && dateRange[0]) {
```

Comment:
Just pasted from this PR https://github.com/teamsempo/SempoBlockchain/pull/410

Reply:
Does that close #410 ?